### PR TITLE
Fix OMS Rate Calculation on AAB-CH

### PR DESCRIPTION
**Pull request by keeysnc**
### Description
Fixed the OMS rate calculation. Previously it was calculating using this formula N/D x 1000. The correct formula for AAB was 1 - (Numerator/Denominator) x 100. 

[MDCT-2680](https://qmacbis.atlassian.net/browse/MDCT-2680)

---
### How to test
1. Create a 2023 child coreset 
2. Select AAB-CH measure
3. Enter N/D/R for OM 
4. Enter the same N/D/R for OMS
5. The ratio calculations should be the same

Before:
Screenshot 2023-07-19 at 12 09 37 PM

After:
Screenshot 2023-07-19 at 12 10 14 PM

---
### Author checklist

- [x] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Review by gmrabian — APPROVED** [review]
looks good! +1 to Aileen's comment

**Review by ailZhou — APPROVED** [review]
LGTM!

**Review by britt-mo — APPROVED** [review]
tested locally, looks good

File: services/ui-src/src/measures/2023/shared/CommonQuestions/PerformanceMeasure/index.tsx [hunk 1]
```diff
@@ -67,6 +67,7 @@ const CategoryNdrSets = ({
   return (
     <>
       {categories.map((item) => {
+        console.log(item);
```
[the remark this answers is not in the document]
Comment (keeysnc) [reply services/ui-src/src/measures/2023/shared/CommonQuestions/PerformanceMeasure/index.tsx]: @ailZhou Thanks! Updated.

**Pull request merged**